float32 x
float32 y
float32 yaw
---
bool result
float32 score